rules/windows/persistence_webshell_detection.toml

```
@@ -21,7 +21,57 @@ license = "Elastic License v2"
 name = "Webshell Detection: Script Process Child of Common Web Processes"
 note = """## Triage and analysis
 
-Detections should be investigated to identify if the activity corresponds to legitimate activity. As this rule detects post-exploitation process activity, investigations into this should be prioritized.
+### Investigating Webshell Detection: Script Process Child of Common Web Processes
+
+Adversaries may backdoor web servers with web shells to establish persistent access to systems. A Web shell is a Web
```

```suggestion
Adversaries may backdoor web servers with web shells to establish persistent access to systems. A web shell is a web
```
better not to capitalize "web"

---

applies elsewhere